api4/cloud.go

```diff
@@ -32,6 +33,9 @@ func getCloudProducts(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	auditRec := c.MakeAuditRecord("getCloudProducts", audit.Fail)
```

Comment:
You have to specifically mark the audit as success `auditRec.Success()` in the end, if not is going to be taken as failure